### DISCUSSION TITLE
Scrolling

### DISCUSSION THREAD

**Discussion by snowboyz0825** [TurboWarp/extensions#1036] [Q&A]
Is there an extension that detects scrolling with the touchpad? I know you could do this with the sensing plus "finger touches" and the turbowarp blocks, but my laptop is not touchscreen and I don't have a mouse. Is there a way to do it with one of the touchpad things with existing extensions or do we have to make a new one

**Comment by HirujaEdurapola** [comment]
How about adding a scrolling bar?

**Comment by LegoBrainBiker** [comment]
I am making this!! #2104
